core/capabilities/compute/monitoring.go

```diff
@@ -0,0 +1,10 @@
+package compute
```

Comment:
@vyzaldysanchez I think this also need to be moved to core/platform/monitoring

Reply:
Can I suggest using core/platform/monitoring/monitoring.go instead? If you look at the imports atm they read as platform.StepRefKey which doesn't say much; monitoring.StepRefKey is better since it'll make it clear it's monitoring related.